kymatio/scattering3d/backend/numpy_backend.py

```diff
@@ -0,0 +1,194 @@
+import numpy as np
+import warnings
+
+BACKEND_NAME = 'numpy'
+from collections import namedtuple
+from scipy.fftpack import fftn, ifftn
+
+
+def _iscomplex(x):
+    return x.dtype == np.complex64 or x.dtype == np.complex128
+
+
+def complex_modulus(input_array):
+    """Computes complex modulus.
```

Comment:
“tensor” should be array and the output is no longer “complex”.

Reply:
Docathon